lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -746,6 +747,7 @@ data ApiTransaction (n :: NetworkDiscriminant) = ApiTransaction
     , withdrawals :: ![ApiWithdrawal n]
     , status :: !(ApiT TxStatus)
     , metadata :: !ApiTxMetadata
+    , deposits :: ![Quantity "lovelace" Natural]
```

Comment:
adjust API types

Reply:
What do you mean ?

Reply:
I meant this adjusted the API types :sweat_smile: ... these comments were meant as a walkthrough